CONTRIBUTING.asciidoc

```diff
@@ -49,89 +49,98 @@ The major technology stacks have their own repository carrying the prefix `devon
 Tools we provide typically have a repository named like the tool they provide (omitting prefixes like `devon[fw]`):
 
 * https://github.com/devonfw/tools-cobigen[CobiGen] (incremental code generator)
-* https://github.com/devonfw/ide[devon-ide] (development environment)
+* https://github.com/devonfw/ide[devonfw IDE] (development environment)
```

Comment:
Thanks for all your improvements, can you write the official name: `devonfw-ide`